cmd/cloud.go

```diff
@@ -55,7 +55,7 @@ var (
 	showCloudLogs = true
 )
 
-//nolint:funlen,gocognit,gocyclo
+//nolint:funlen,gocognit,gocyclo,cyclop
```

Comment:
```suggestion
//nolint:funlen,gocognit,cyclop
```
gocyclo has been disabled

Reply:
why didn't nolintlint told me that then :thinking:

Reply:
There is an open questions for it https://github.com/golangci/golangci-lint/discussions/2395